buildconfig/download_win_prebuilt.py

```
@@ -78,12 +78,12 @@ def get_urls(x86=True, x64=True):
     url_sha1 = []
     url_sha1.extend([
         [
-        'https://github.com/libsdl-org/SDL/releases/download/release-2.32.8/SDL2-devel-2.32.8-VC.zip',
-        'ec8bbf2a2d2bf754fe04b6ffc647f78aa7f4c2a1',
+        'https://github.com/libsdl-org/SDL/releases/download/release-2.32.10/SDL2-devel-2.32.10-VC.zip',
```

Flake8 (7.2.0)

[error] continuation line missing indentation or outdented

(E122)